### PR TITLE
fix: weird color formats (#8)

### LINKED ISSUE #8: Weird ouput
Hey, I'm excited about this project. Thank you for your fantastic stuff, @egoist.

I encounter this output (when I try installing dum) both in st (suckless terminal) and alacrity.
```
 bagher ~ $ curl -sSL https://bina.egoist.sh/egoist/dum | sh

\x1B[33m  ==>\x1B[39m Using auto generated config because egoist/dum doesn't have a bina.json file in its release
\x1B[33m  ==>\x1B[39m This might not work for some projects
\x1B[94m  ==>\x1B[39m Resolved version latest to v0.1.12
\x1B[94m  ==>\x1B[39m Downloading asset for linux amd64
```
I'd like to contribute to this project, it'd be really good if you can give me a hint about this output.

### PR DESCRIPTION
**Pull request by Aslemammad**
An attempt to fix #8.

Special thanks to @egoist for the hints.

### CODE REVIEW

**Comment by Aslemammad** [comment]
Alacrity:
image

St:
image

I hope these are the correct formats.

**Comment by egoist** [comment]
weird it didn't download the file.

M. Bagher Abiat ***@***.***>于2021年12月6日 周一下午5:51写道：

**Comment by Aslemammad** [comment]
The diff code is not related to that I guess. is the problem from vercel (`Authorization required to deploy.`)?

File: make_sh.ts [hunk 1]
```diff
@@ -59,16 +67,17 @@ export const makeInstallerScript = ({
   }
   
   log_info() {
-    printf "\\x1B[94m  ==>\\x1B[39m $@\n"
+    printf "\${BRIGHT_BLUE}  ==>\${NC} $@\n"
+    exit 0
```
Comment (egoist) [inline make_sh.ts]: should remove `exit 0`

Comment (Aslemammad) [reply make_sh.ts]: Oh no.

File: make_sh.ts [hunk 2]
```diff
@@ -50,6 +53,11 @@ export const makeInstallerScript = ({
   return `
   #!/bin/sh
 
+  NC='\\033[0m' # No Color
+  BRIGHT_BLUE='\\033[0;94m'
+  YELLOW='\\033[0;33m'
+  RED='\\033[0;31m'
+
```
Comment (egoist) [inline make_sh.ts]: let's move this below `set -e`

Comment (Aslemammad) [reply make_sh.ts]: Sure

**Comment by Aslemammad** [comment]
Done! How about now?

**Pull request merged**